src/components/notifications/Bark.vue

```diff
@@ -12,4 +12,109 @@
             >{{ $t("here") }}</a>
         </i18n-t>
     </div>
+    <div class="mb-3">
+        <label for="Bark Group" class="form-label">{{ $t("Bark Group") }}</label>
+        <input id="Bark Group" v-model="$parent.notification.barkGroup" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="Bark Sound" class="form-label">{{ $t("Bark Sound") }}</label>
+        <select id="Bark Sound" v-model="$parent.notification.barkSound" class="form-select" required>
+            <option value="alarm">
```

Comment:
Is there a way that these options could be added programatically, i.e from an array. Or at least each option only be on one line. It just looks a bit long as it is.

Reply:
yes，the names is in [this](https://github.com/Finb/Bark/tree/master/Sounds)

I don't know how to write arrays in `.vue` file

Reply:
Here is an example:
https://github.com/louislam/uptime-kuma/blob/f8f19d8dc5cd668665f4a4a6d3b46782fe556383/src/components/NotificationDialog.vue#L15-L17

https://github.com/louislam/uptime-kuma/blob/f8f19d8dc5cd668665f4a4a6d3b46782fe556383/src/components/NotificationDialog.vue#L82-L98

You can use `v-for` to itterate over the array and then just put the array in `data`.

Another option if you don't want to do this would be to just put each on one line i.e. 
```
<option value="alarm"> alarm </option>
```

Just something to shorten it up a bit as currently it is a too many lines long.